src/runtime/gateways/cloud/gateway.ts

```
@@ -508,91 +581,78 @@ export class FireproofCloudGateway implements SerdeGateway {
+    // console.log("getCloudConnectionItem", uri);
+    let protocol = uri.getParam("protocol", "https");
+    switch (protocol) {
+      case "wss":
+        protocol = "https";
+        break;
+      case "ws":
+        protocol = "http";
+        break;
+      case "http":
+        break;
+      case "https":
```

Biome (1.9.4)

[error] Useless case clause.

because the default clause is present:

Unsafe fix: Remove the useless case.


(lint/complexity/noUselessSwitchCase)